plugins/inputs/ah_wireless/ah_wireless_defines.go

```diff
@@ -487,6 +489,7 @@ type utilization_data struct {
 	noise_min			int16
 	noise_max			int16
 	noise_avg			int16
+	pad         [2]byte
```

Comment:
Padding is not required. Since this structure is not used in ioctl. It is only used in telegraf internal.